assets/queries/k8s/statefulset_without_service_name/query.rego

```diff
@@ -1,10 +1,18 @@
-package Cx
+package Cx  
```

Comment:
Please, also update the query `kics\assets\queries\terraform\kubernetes\statefulset_without_service_name`

If possible, also improve the query description, please